### DISCUSSION TITLE
Does 再 in this context have the same meaning of "again"?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [rodrigo-morales-1/subtitles-npcr#1087] [Q&A] [answered]
https://github.com/rdrg109/subtitles-npcr/blob/b5d415b7f3aaa0fc4e31696621a845c1f0287a6a/sentences/45-text-1.yaml#L57-L61

@fhxshrs I think you should know this one. Please ping me when answering, so that I can get an email when you answer.

**Comment by fhxshrs** [comment, marked as answer]
@rdrg109 是的，有 "again" 的意思，不过既然这已经是钱经理和马大为第二次谈求职问题了，所以英语翻译为“进一步讨论”我觉得也没什么问题，你觉得呢？

**Reply by rodrigo-morales-1** [reply]
@fhxshrs I don't see a problem either. This time they will discuss for the second time and they will probably reach a conclusion, so the current translation "to further discuss" makes sense here.

I was asking because I wanted to show a direct translation for 再 to comply with [guideline no. 3](https://github.com/rdrg109/subtitles-npcr/blob/ec84d0c7642202f2c2314e16ed48a989f8e4e5ab/docs/guidelines.org#3-show-direct-translations-as-much-as-possible), so that Chinese learners know the meaning of 再 in this context.

So, we have two possible translations:
* to discuss **again** questions we have on your application.
  * Pros: we show a direct translation for 再
  * Cons: this express less meaning than "further discuss"
* to **further** discuss questions we have on your application.
  * Pros: this express deeper meaning than "discuss again"
  * Cons: we don't show a direct translation for 再

I think we should choose the first one because it is more beneficial for Chinese learners. In, 4d007f0d7f48ebda5b35791f3cf6b5f2267f794b I edited that sentence to show the first translation. I should have replied to this message before introducing those changes.

**Reply by fhxshrs** [reply]
Thank you for your explanation!
